### PR TITLE
Add popular NPM keywords [ci skip][skip ci]

### PR DESCRIPTION
**Pull request by cboulanger**
Adds some popular fitting NPM keywords so that qooxdoo shows up when people browse them

### CODE REVIEW

**Review by hkollmann — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -6,7 +6,18 @@
   "keywords": [
     "qooxdoo",
     "compiler",
-    "cli"
+    "cli",
+    "framework",
+    "front-end",
+    "mobile",
+    "back-end",
```
Comment (derrell) [inline package.json:13]: backend (vs back-end) and frontend are common spellings. Maybe add them without the dashes as well?

**Review by derrell — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -6,7 +6,18 @@
   "keywords": [
     "qooxdoo",
     "compiler",
-    "cli"
+    "cli",
+    "framework",
+    "front-end",
```
Comment (oetiker) [inline package.json:11]:
```suggestion
    "front-end",
    "fontend",
```

File: package.json (hunk 1 above)
Comment (oetiker) [inline package.json:13]:
```suggestion
    "back-end",
    "backend",
```

Comment (cboulanger) [reply package.json:13]: The main NPM keywords are actually with the dashes, but sure, we can add the other spelling as well.

**Review by johnspackman — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Pull request merged**